screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionViewModel.kt

```diff
@@ -684,7 +685,7 @@ class EditTransactionViewModel @Inject constructor(
                     dateTime = when {
                         loadedTransaction().dateTime == null &&
                                 dueDate.value == null -> {
-                            timeNowUTC()
+                            timeNowLocal()
```

Comment:
Are you sure that this is sufficient to fix the issue? Can you attach a screen recording where you test with:
- UTC+8
- UTC+0
- UTC-8

Reply:
yes sure.

Reply:
https://github.com/Ivy-Apps/ivy-wallet/assets/42218722/e35ae892-22ca-40d9-9dae-8259561a3ba2


https://github.com/Ivy-Apps/ivy-wallet/assets/42218722/3b1a8fd0-d1b5-44c0-a59a-6a0f10d73783


https://github.com/Ivy-Apps/ivy-wallet/assets/42218722/340aaf0b-5876-4005-90ed-7ebbc8dece91

Reply:
Nice!